django/db/backends/sqlite3/operations.py

```diff
@@ -237,7 +239,8 @@ def convert_timefield_value(self, value, expression, connection):
     def convert_decimalfield_value(self, value, expression, connection):
         if value is not None:
             value = expression.output_field.format_number(value)
-            value = backend_utils.typecast_decimal(value)
+            # Value is not converted to Decimal here as it will be converted
```

Comment:
Just wondering, did you abandon [the idea](https://github.com/django/django/pull/8854#issuecomment-321002518) to deprecate `convert_value()` or might this be changed back after that?